shop/shop/src/main/java/ro/msg/learning/shop/converter/HttpMessageConverterCSV.java

```diff
@@ -0,0 +1,55 @@
+package ro.msg.learning.shop.converter;
```

Comment:
all other packages are using plurals for naming. change this as well to "converters"